crates/uv-git-types/src/lib.rs

```diff
@@ -120,7 +187,8 @@ impl TryFrom<DisplaySafeUrl> for GitUrl {
             url.set_path(&prefix);
         }
 
-        Self::from_reference(url, reference)
+        // TODO(samypr100): GitLfs::from_env() for now unless we want to support parsing lfs=true
+        Self::from_reference(url, reference, GitLfs::from_env())
```

Comment:
This feels slightly dangerous because it changes all instances of this type depending on the environment